v2/plugin/collector.go

```diff
@@ -93,4 +93,9 @@ type CollectorDefinition interface {
 
 	// Allow metrics values not only on leaves but at any namespace level
 	AllowValuesAtAnyNamespaceLevel()
+
+	// Set global namespace prefix for all metrics (used by aggregator collector with metrics with different roots name).
+	// removePrefixFromOutput set to true removes global prefix from all namespaces when result is to be sent to agent.
```

Comment:
do you see any use case when we might _not_ want this set to true?

Reply:
currently no, but I would leave it

Reply:
ok